tests/drivers/entropy/api/Kconfig.mcux

```diff
@@ -0,0 +1,9 @@
+# Copyright NXP 2022
```

Comment:
This should be selected in the boards folder config file.

Reply:
I am not sure if I agree, because this is needed because of using the caam, it's not specific to a board

Reply:
I am wondering about future SoC variations from NXP that may not need this Kconfig.

Reply:
okay.... but what about future variations that would need this kconfig because they have a CAAM... you would need to remember to add a board overlay on this test for every evk with a caam.... it just seems easier to have the kconfig selected automatically just depending of if the SOC has a caam